docs/user_guide.md

```diff
@@ -24,80 +24,92 @@ $ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10
 Argument List:
 - config: Mandatory. The path to the CSV config file.
 - output: Optional. The path to the dashboard generated. If not provided, it will be generated in the current directory.
-- since : Optional. start date of analysis. Format: dd/MM/yyyy
-- until : Optional. end date of analysis. Format: dd/MM/yyyy
+- since : Optional. start date of analysis. Format: `DD/MM/YYYY`
+- until : Optional. end date of analysis. Format: `DD/MM/YYYY`
 
 ```
 Note:
 The contribution calculation is base on the daily commits made within 00:00 to 23:59 in GMT+8.
 ```
 
 ## CSV Config File
-The CSV Config files control the list of target repositories. It also contains a white list of authors(if the author is listed in the CSV, his/her contribution will be ignored by the analyzer.)
-[Sample_full.csv](../sample_full.csv) in root is an example CSV config file. It should contain the following columns:
+The CSV Config files control the list of repositories and the target author.
+[Sample_full.csv](../sample_full.csv) is an example configuration file setup. It should contain the following columns:
 
 Column Name | Explanation
 ----------- | -----------
 Organization | Organization of the target repository
 Repository | Name of the target repository
 branch | Target branch
-StudentX's Github ID | Author's Github ID.
-StudentX's Display Name | Optional Field. The value of this field, if not empty,will be displayed in the dashboard instead of author's Github ID.
-StudentX's Local Author Name | Detailed explanation below
+ContributorX's Github ID | Author's Github ID.
```

Comment:
Should we standardize between Contributor and Author? 
```Author``` is a POJO we use internally.

Reply:
Also, we can drop the X as #130 has changed the CSV format to use one ```Author``` per row.